galasa-managers-parent/galasa-managers-logging-parent/dev.galasa.elasticlog.manager/src/main/java/dev/galasa/elasticlog/internal/ElasticLogManagerImpl.java

```diff
@@ -205,18 +209,29 @@ public void testClassResult(@NotNull String finalResult, Throwable finalExceptio
 			client.setTrustingSSLContext();
 			client.addOkResponseCode(201);
 			client.setURI(new URI(endpoint));
-
-			//Send document to index
-			client.postText(index + "/_doc", request);
-			logger.info("Run successfully logged to Elastic index " + index);
+			
+			int statusCode = client.postJson(index + "/_doc", json).getStatusCode();
+			
+			//Send document to index and check response code
+			if(statusCode != 201 && statusCode != 200){
+			   logger.error("Error logging to Elastic index " + index + ". Status Code: " + statusCode);
```

Comment:
if you are going to log this,  you will need to log any error message that is returned